server/src/controllers/game.controller.js

```diff
@@ -80,13 +83,16 @@ class GameController {
 
   async move(req, res) {
     try {
-      const { name, move, gameId, permission } = req.body;
+      const { name, move, gameId, permission, player_id, board_state } = req.body;
       // do validations
       const game = retrieve(gameId);
       if (!game) return res.status(400).json({ message: "no such game" });
 
       //cache moves or save to db later
 
+      //No one has created a function to add game to db
+      //Here i use a dummy data i added via postman
+      await saveMoveToDb({player_id: 1, board_state: 4, gameId: 11})
```

Comment:
What you are passing here doesn't correspond to the parameter in the function definition